src/main/java/org/littleshoot/proxy/mitm/SubjectAlternativeNameHolder.java

```diff
@@ -58,4 +61,17 @@ private ASN1Encodable parseGeneralName(List<?> nameEntry) {
         }
         throw new IllegalArgumentException(String.valueOf(nameEntry));
     }
+
+    public String getAlternativesIdentifier(String commonName) {
+        if (commonName == null) {
+            throw new IllegalArgumentException(
+                "Error, 'commonName' is not allowed to be null!");
+        }
+        TreeSet<String> alternativesIdentifier = new TreeSet<>();
+        for (ASN1Encodable san : sans) {
+            alternativesIdentifier.add(san.toString());
+        }
+        return alternativesIdentifier.isEmpty() ? commonName : commonName + ALTERNATIVES_IDENTIFIER_DELIMITER +
```

Comment:
let's avoid this check by adding `commonName` to `alternativesIdentifier`:

```java
alternativesIdentifier.add(commonName);
...
return Joiner.on(ALTERNATIVES_IDENTIFIER_DELIMITER).join(alternativesIdentifier);
```

Reply:
Fixed